docs/usage-tui.md

```diff
@@ -0,0 +1,207 @@
+- ADA coin amounts (`1.5ADA`, see below)
+- Cardano addresses without quotes (`sxtitePxjp5dcfm1u8gWgDBGMCEZMhGa6kUPu8VHhqpCtBDPExrJTTCCUHKkyEJSgjb41JT5Tfh1QXb7uUpgjyBKMw`)
+- Task ids (`<1>`, see below)
+- File paths, either relative or absolute, without quotes (`./file.txt`, `/etc/file.txt`)
+
+Let's first cover `Knit` commands that are not related to cryptocurrency. There are not many:
+
+- `L`: create a list of things. Call it with any number of arguments, which may be of different
+    types:
+
+ ```
+ L 1 2 3 "a" "b" "c"
+ ```
+
+- `help` and `logs`: switch to Help and Logs tabs respectively
+- `print`: print an argument of any type
+- `sleep`: delay execution for a given number of microseconds
+- `not`, `true` and `false`: boolean negation and two boolean literals
+
+Aside from that you can enter multiple commands in one input. To do so, separate the commands with
+`;`.  REPL widget input line also allows you to break lines for visual stylization of commands. If
+you enter a backslash (`\ `) and press the Enter key, a new input line will appear.
+
+All `Knit` commands are executed asynchronously, giving you back the control of REPL right after you
+press Enter. Ariadne has a task manager for these background commands.  When you run some
+long-running task, its id (a number in angle brackets like `<1>`) and a string `"Waiting for
+result"` will appear. To cancel a command, use `kill` command like so:
+
+    kill <1>
+
+You can also make some commands depend on others by using `wait` command with a specific task id.
+
+### User interface and Knit integration
+
+Some of the `Knit` functions may access the internal state of Ariadne UI &mdash; namely, currently
+selected wallet, account or address. This allows you to omit the wallet name when creating a new
+account using `new-account` command &mdash; it will use the currently selected one. There is also a
+`copy-selection` command, which will copy the name of selected item (wallet, account or address) to
+the system clipboard.
+
+### Cardano-related commands
+
+Finally, to the commands which will help you manage your `ADA` wallet. Each command will be
+annotated with its arguments and their types. Optional arguments will be marked with "`?`".
+
+- `new-wallet pass: String? name: String? entropy-size: Int?`: create a new wallet with a
+    passphrase, name and size of entropy. Size of entropy influences security of your wallet and the
+    length of the mnemonic. Default value is 16 bytes, leading to mnemonics of 13 words.
+- `new-account wallet: String or Word name: String?`: create a new account in the specified or
+    the currently selected wallet. You can specify wallet either by its name or by its (zero-based)
+    index in the wallet tree.
+- `new-address wallet: String or Word account: String or Word pass: String?`: create a new address
+    with an optional passphrase. This command will use either the currently selected account or the
```

Comment:
It makes an impression that the passphrase will be used to encrypt the address, but it's used to decrypt the wallet key. I'd rephrase it somehow. Passphrase must match one used for the wallet creation.

Reply:
I understood it the wrong way, so I wrote it the wrong way :) Changed.